android/quest/src/main/assets/sync_by_organization_publisher.json

```diff
@@ -0,0 +1,80 @@
+[
```

Comment:
change file name to sync-config and move it to config/app folder. We would have only one sync config and eventually in composition.. so its good to have generic name